packages/core/src/lib/TwaGenerator.ts

```diff
@@ -224,54 +225,67 @@ export class TwaGenerator {
    * @param {String} targetDirectory the directory where the project will be created
    * @param {Object} twaManifest configurations values for the project.
    */
-  async createTwaProject(targetDirectory: string, twaManifest: TwaManifest): Promise<void> {
+  async createTwaProject(targetDirectory: string, twaManifest: TwaManifest,
+      reportProgress: twaGeneratorProgress = noOpProgress): Promise<void> {
```

Comment:
So we use the default value here when calling `createTwaProject` from `init.ts` - why don't we want to show the progress bar in that case?

Reply:
`init` is invoking the method from `shared.ts`, so it will show the progress bar. `TwaGenerator` is part of `core` and applications using it probably don't want the progress bar.